PS-3571 Allow Catalogue Query Order By

wbelhomsi opened the pull request:
We want to order by different things when querying the catalogue other than just name.
To do that we want to extend the shortcode functionality to allow it to have two params
order_by and order_direction which will server as the order fields to the gql query
this will allow us to sort by date for example on our clients pages

Commit:
PS-3571 Allow Catalogue Query Order By

We want to order by different things when querying the catalogue other than just name.
To do that we want to extend the shortcode functionality to allow it to have two params
order_by and order_direction which will server as the order fields to the gql query
this will allow us to sort by date for example on our clients pages

gsavage commented:
This PR relates to Jira ticket PS-3571 - see https://administrate.atlassian.net/browse/PS-3571

jadkhateradmin committed:
PS-3571 Refactor passing config params to bundles shortcode html templates, ajax call and php methods

In this commit we are refactoring how we are passing shortcode config params to the html template, ajax request, callback php methods making updating or adding params a more stream lined proccess for the bundles shortcode plugin.
